setup.py

```diff
@@ -1,34 +1,70 @@
 """
-Setup module for trustymail
+setup module for trustymail
 
 Based on:
 
-- https://github.com/dhs-ncats/pshtt
```

Comment:
Personally, I would probably leave the link back to pshtt.

Reply:
I agree with @IanLee1521.